src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/Tensor.cs

```diff
@@ -671,5 +682,18 @@ public string ToString(params ReadOnlySpan<nint> maximumLengths)
             sb.AppendLine("}");
             return sb.ToString();
         }
+
+        /// <summary>
+        /// Pins and gets a <see cref="MemoryHandle"/> to the backing memory.
+        /// </summary>
+        /// <returns><see cref="MemoryHandle"/></returns>
+        public MemoryHandle GetPinnedHandle()
```

Comment:
>GetPinnedHandle [](http://example.com/codeflow?start=28&length=15)

Is it possible to unpin the tensor after it is pinned? just curious to know :-)

Reply:
Yeah. If you dispose of the MemoryHandle it will dispose of the GCHandle and unpin it.